### PR TITLE
List Webchain (WEB)

### PR DESCRIPTION
**Pull request by khaledalam**
- Official project URL: https://webchain.network
- Official block explorer URL: https://explorer.webchain.network

### CODE REVIEW

**Comment by khaledalam** [comment]
@blabno please close #1779 after merge this PR.

File: assets/src/main/java/bisq/asset/coins/Webchain.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class Webchain extends Coin {
+
+    public Webchain() {
+        super("Webchain", "WEB", new RegexAddressValidator("^0x[0-9a-fA-F]{40}$"));
+    }
+}
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Webchain.java]: Add newline at the end of file

File: assets/src/test/java/bisq/asset/coins/WebchainTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @Test
+    public void testInvalidAddresses() {
+        assertInvalidAddress("");
+        assertInvalidAddress("8d1ba0497c3e3db17143604ab7f5e93a3cbac68b");
+        assertInvalidAddress("0x8d1ba0497c3e3db17143604ab7f5e93a3cbac68");
+        assertInvalidAddress("0x8d1ba0497c3e3db17143604ab7f5e93a3cbac68k");
+        assertInvalidAddress("098d1ba0497c3e3db17143604ab7f5e93a3cbac68b");
+        assertInvalidAddress("098d1ba0497c3e3db17143604ab7f5e93a3cbac68b");
+    }
+}
```
Comment (blabno) [inline assets/src/test/java/bisq/asset/coins/WebchainTest.java]: Add newline at the end of file

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments
But you're almost there

File: assets/src/main/java/bisq/asset/coins/Webchain.java (hunk 1 above)
Comment (khaledalam) [reply assets/src/main/java/bisq/asset/coins/Webchain.java]: @blabno i created new PR and added newline at the end of file here: #2167 but seems same changes.

**Pull request closed without merging**